configure/recipes/timescale.rb

```diff
@@ -0,0 +1,47 @@
+unless node.attribute?(:ec2)
+  # Copy config files
+  cookbook_file "/etc/postgresql/12/main/pg_hba.conf" do
+    source "pg_hba.conf"
+    owner "root"
+    group "root"
+    mode "0644"
+  end
+  cookbook_file "/etc/postgresql/12/main/postgresql.conf" do
+    source "postgresql.conf"
+    owner "root"
+    group "root"
+    mode "0644"
+  end
+
+  # Tune pgsql for timescale
+  execute "tune-and-restart-pgsql" do
+    command "sudo timescaledb-tune -yes -quiet; sudo service postgresql restart"
+    user "root"
+  end
+
+  # restart pgsql
+  service "pgsql" do
+    supports status: true, restart: true, reload: true
+    action %i{stop disable}
+  end
+
+  # Create pgsql user 'root' if it doesn't exist
+  execute "pgsql-default-user" do
+    command "psql -c \"CREATE USER root WITH PASSWORD '#{node["pgsql"]["root_password"]}' SUPERUSER\""
```

Comment:
I know you added this config to the environment file, but we should also add it as a default attribute, otherwise deploying these changes on existing nodes will throw exceptions if their manual attributes haven't been modified.

Add to the bottom of `base/attributes/default.rb`.